sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/RspecStrings.resx

```diff
@@ -11234,7 +11234,7 @@
     <value>Critical Vulnerability</value>
   </data>
   <data name="S5547_Description" xml:space="preserve">
-    <value>Strong cipher algorithms are cryptographic systems resistant to cryptanalysis, they are not vulnerable to well-known attacks like brute force attacks for example.</value>
+    <value>Strong cipher algorithms are cryptographic systems resistant to cryptanalysis, they are not vulnerable to well-known attacks like brute force attacks for example. </value>
```

Comment:
nitpick: unrelated change, but I'll merge anyway

Reply:
ah, I just checked the HTML files and missed this one :( Where's the space coming from? I wanted to fix it in Jira before rspec update, but I didn't find it.